docs/user-manual/settings.md

```diff
@@ -21,6 +21,15 @@ time for the convenience of your station staff. You can also set the day of the
 week that you wish to start your station's weekly schedule on, which defaults
 to Sunday.
 
+:::note
+
+The **Station Timezone** setting can not be modified here and is set in the master
+config file. Please consult
+[this admin manual section](../admin-manual/configuration/#general) on how to
+change it.
```

Comment:
```suggestion
The **Station Timezone** setting can not be modified on this page. It is set in the [configuration file](../admin-manual/configuration/#general).
```

Somewhat unrelated to this PR, but we'll let it slip :wink: .

Reply:
Sorry, i have a habit to update the page i am working on, if i see stuff is outdated. Can't really help it :-)

Reply:
Great that you did!